src/app/lib/utilities/parseAssetData/index.js

```diff
@@ -1,12 +1,26 @@
 import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
 
 const getISOStringDate = date => new Date(date).toISOString();
 
 export const getArticleId = path(['metadata', 'id']);
 
 export const getHeadline = path(['promo', 'headlines', 'seoHeadline']);
 
-export const getSummary = path(['promo', 'summary']);
+export const getSummary = pathOr('', [
```

Comment:
In the issue, there's a requirement `Update fixture data.` - to do this, you'll need to go through all local Optimo Article assets `/data/<service>/articles/<asset id>.json` and replace the promo.summary with one that has the same text, but has this block structure. 

Otherwise, when you visit an Optimo page locally, you won't get the correct summary field being used in the description meta tag.